parsec/core/gui/greet_user_widget.py

```diff
@@ -549,7 +561,12 @@ def __init__(self, core, jobs_ctx, token):
         self.greeter_job = None
         self.greeter_success.connect(self._on_greeter_success)
         self.greeter_error.connect(self._on_greeter_error)
+        self.organization_status_job = None
+        self.outsider_enabled = False
+        self.organization_status_success.connect(self._on_organization_status_success)
+        self.organization_status_error.connect(self._on_organization_status_error)
         self._run_greeter()
+        self._run_organization_status()
```

Comment:
It's dangerous to run two jobs concurrently here (`_run_greeter` and `_run_organization_status`), we have no strong guarantee organization status will be done by the time we will need `self.outsider_enabled`...

A better solution would be to add this to the `BackendAuthenticatedConn._manager_connect`, so fetching organization status would become part of the initialization process:
```python

    async def _manager_connect(self):
        async with self._acquire_transport(ignore_status=True, force_fresh=True) as transport:
            self.set_status(BackendConnStatus.INITIALIZING)
            self._backend_connection_failures = 0
            logger.info("Backend online")

            rep = await cmds.organization_status(transport)
            if rep["status"] != "ok":
                # We may want to do a better check on the status values (which are expected which one are not and should trigger the unhandled exception logging)
                raise BackendConnectionRefused("huho !")
            self.set_organization_status(OrganizationStatus(expiration_date=rep["expiration_date"], outsider_enabled=rep["outsider_enabled"]))

            await cmds.events_subscribe(transport)
            # BTW we should be checking events_subscribe's return value too here !
            [...]
```
This way we would always have the organization status available from a synchronous function !
```
org_status = logged_core.get_organization_status()  # No async means no job schedule from Qt \o/
if org_status.outsider_enabled:
   [...]
```

Of course for the function to be sync, there should be a default value for `organization_status`. We could store the last received value in the local storage (considering it is unlikely to change), but I think we can just create a dummy default (`outsider_enabled=False`, `expiration_date=None`) which would be close enough for now and save us some tedious work ;-)

On the same idea, we could add a organization_status_updated event coming from the backend to notify the clients, but given this is a very unlikely event I think it's just better to ignore this for now